package-lock.json

```diff
@@ -1,13131 +1,8 @@
 {
   "name": "webdav",
   "version": "4.2.1",
-  "lockfileVersion": 2,
```

Comment:
Can you please revert the entire package-lock change? It's far too large a change for no dependencies having been updated.

Reply:
If you update from master this is already handled - the lock file was an old version and now it should more closely match what you submitted.